javagen/src/main/java/com/azure/autorest/Javagen.java

```diff
@@ -125,6 +129,13 @@ protected NodeTuple representJavaBeanProperty(Object javaBean, Property property
                 }
             }
 
+            if (settings.isGenerateLLCSamples()) {
+                client.getServiceClient().getMethodGroupClients()
+                        .forEach(c -> c.getClientMethods().stream()
+                        .filter(m -> m.getType() == SimpleSyncRestResponse || m.getType() == PagingSync)
+                        .forEach(m -> javaPackage.addProtocolExamples(m, c, builderName)));
```

Comment:
ServiceClient can contain some ClientMethod as well (if these APIs does not have operation group), though you may delay this until you encounter one.

Reply:
I created an [issue](https://github.com/Azure/autorest.java/issues/1184). Let's do it in another PR.